kubernetes/build.gradle

```diff
@@ -18,6 +18,24 @@ ext {
   outputFilePython = file("$buildDir/dist/Dockerfile-Python")
   outputFileR = file("$buildDir/dist/Dockerfile-R")
   outputFileScala = file("$buildDir/dist/Dockerfile-Scala")
+  outputFileExternalBackend = file("$buildDir/dist/Dockerfile-External-backend")
+}
+
+task createExternalBackendDockerfile(type: Dockerfile) {
+  // Based on h2o-3/docker/public/Dockerfile-h2o-release
+  // After official H2O image is out, remove duplication as mentioned in https://0xdata.atlassian.net/browse/SW-2294
+  destFile = outputFileExternalBackend
+  from "openjdk:11"
+  runCommand "mkdir -p /opt/h2oai/h2o-3/"
+  copyFile("h2o.jar", "/opt/h2oai/h2o-3/")
+  copyFile("sparkling-water-assembly-extensions_2.11-${version}-all.jar", "/opt/h2oai/h2o-3/")
+  defaultCommand(
+    "java",
+    "-cp",
+    "/opt/h2oai/h2o-3/sparkling-water-assembly-extensions_2.11-${version}-all.jar:/opt/h2oai/h2o-3/h2o.jar",
+    "-XX:+UseContainerSupport",
```

Comment:
Isn't this option enabled for java 11 by default?

Reply:
For this, you need to ask @Pscheidl. I'm just copying the Dockerfile statements for now.

Once H2O official docker image is released, I will change this part of code to extend the h2o image

Reply:
Did some googling, yup it is default. However even though it might not be necessary, I propose mimicking what H2O has to have same images